frontend/src/metabase/query_builder/components/filters/FilterPopover.tsx

```diff
@@ -48,7 +48,6 @@ type Props = {
   dateShortcutOptions?: DateShortcutOptions;
   showCustom?: boolean;
   isNew?: boolean;
-  isSidebar?: boolean;
```

Comment:
The filter sidebar was just a wrapper around the filter popover with a bunch of props drilled down into various components to trigger layout differences. We can remove all those conditionals and simplify a bunch of these components now.